pom.xml

```diff
@@ -31,14 +31,17 @@ limitations under the License.
   <description>The Accumulo Recipes project aims to provide reusable services for architectures utilizing Apache
         Accumulo.</description>
   <modules>
+    <module>minicluster-test-driver</module>
```

Comment:
Oh... i think my reason was that I didn't want to add a test dependency on the mini cluster if there wasn't a need, but maybe that was overkill.